gemini/context-caching/intro_context_caching.ipynb

```diff
@@ -31,8 +31,6 @@
       "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        contents=[
</pre><p>See you already updated another file. </p><p>contents has been moved to CreateCachedContentConfig :</p><p>cached_content = client.caches.create(</p><p>    model=MODEL_ID,</p><p>    config=CreateCachedContentConfig(</p><p>        contents=[</p><p>            Content(</p><p>            role="user",</p><p>            parts=pdf_parts,</p><p>            )</p><p>        ],</p><p>        system_instruction=system_instruction,</p><p>        ttl="3600s",</p><p>    ),</p><p>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1601/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Fixed